pandas/io/pickle.py

```diff
@@ -72,6 +101,40 @@ def read_pickle(path, compression='infer'):
     Returns
     -------
     unpickled : type of object stored in file
+
+    See Also
+    --------
+    DataFrame.to_pickle : Pickle (serialize) DataFrame object to input file
+        path.
+    Series.to_pickle : Pickle (serialize) Series object to input file path.
+    read_hdf : read from the store, close it if we opened it.
```

Comment:
I think the explanation about closing the file is too detailed for the See Also explanation. Maybe something like "Read HDF5 file into a DataFrame" ?

Reply:
amended 👍